requirements.txt

```diff
@@ -1,6 +1,7 @@
-boxsdk==1.5.3
 celery==4.0.2
 django==1.10.5
+django-celery-beat==1.0.1
 django-cors-headers==2.0.2
 djangorestframework==3.4.7
+wget==3.2
```

Comment:
is this needed? Seems like you are using requests library instead (which sounds good)